hiro/hiro_utils.py

```diff
@@ -107,14 +107,19 @@ def sample(self):
         )
 
 class SubgoalActionSpace(object):
-    def __init__(self, dim):
-        limits = np.array([-10, -10, -0.5, -1, -1, -1, -1,
-                    -0.5, -0.3, -0.5, -0.3, -0.5, -0.3, -0.5, -0.3])
-        self.shape = (dim,1)
-        self.low = limits[:dim]
-        self.high = -self.low
+    def __init__(self, dim, limits=np.array([-10, -10, -0.5, -1, -1, -1, -1,
```

Comment:
yeah this will do ... but might be better to have a Builder class/function to read in from a file or something.

Reply:
I'm thinking that this is enough, maybe in a separate pr?